docker/README.md

```diff
@@ -324,6 +362,29 @@ To stop the MMS running inside the docker instance, run the following
 ```bash
 $ docker exec mms bash -c "mxnet-model-server.sh stop"
 ```
+
+## Usage help for MMS
+```bash
+$ docker exec mms bash -c "mxnet-model-server.sh help"
+Usage:
+
+/mxnet_model_server/mxnet-model-server.sh [start | stop | restart | help] [--mms-config <MMS config file>]
+
+start        : Start a new instance of MXNet model server.
+stop         : Stop the current running instance of MXNet model server
+restart      : Restarts all the MXNet Model Server worker instances.
+help         : Usage help for /mxnet_model_server/mxnet-model-server.sh
+--mms-config : Location pointing to the MXNet model server configuration file.
+To start the MXNet model server, run
+/mxnet_model_server/mxnet-model-server.sh start --mms-config <path-to-config-file\>
+
+To stop the running instance of MXNet model server, run
+/mxnet_model_server/mxnet-model-server.sh stop
+
+To restart the running instance of MXNet model server, run
+/mxnet_model_server/mxnet-model-server.sh restart --mms-config <path-to-config-file\>
+```
+
 ## Debugging or logging into the running docker instance
 To debug the docker instance further you could run the following commands
```

Comment:
Add a colon, and for the commands, put them in bash blocks.

Reply:
For the SSL section I'd reorder it - it isn't clear if you're creating the cert in the container or not... so maybe you run it, then create the cert, then update nginx with the cert file locations.... but then I think you'd lose the cert once the image was stopped.... so I question whether or not these SSL instructions are really fleshed out or not...

Reply:
I am rewriting the whole SSL part.